python/lsst/afw/cameraGeom/utils.py

```diff
@@ -603,16 +716,26 @@ def showAmp(amp, imageSource=FakeImageDataSource(isTrimmed=False), display=None,
 
 def showCcd(ccd, imageSource=FakeImageDataSource(), display=None, frame=None, overlay=True,
             imageFactory=afwImage.ImageF, binSize=1, inCameraCoords=False):
-    """!Show a CCD on display
-
-    @param[in] ccd  Detector to use in display
-    @param[in] imageSource  Source for producing images to display.  Must have a getCcdImage method.
-    @param[in] display image display to use
-    @param[in] frame  frame ID on which to display
-    @param[in] overlay  Show amp bounding boxes on the displayed image?
-    @param[in] imageFactory  The image factory to use in generating the images.
-    @param[in] binSize  Binning factor
-    @param[in] inCameraCoords  Show the Detector in camera coordinates?
+    """Show a CCD on display.
+
+    Parameters
+    ----------
+    ccd : `lsst.afw.cameraGeom.Detector`
+        Detector to use in display.
+    imageSource : `class` with getCcdImage method
+        Source for producing images to display.  Must have a getCcdImage method.
+    display : `lsst.afw.display`
+        image display to use.
+    frame :
```

Comment:
Type?

Reply:
I have no idea. The docs for `_getDisplayFromDisplayOrFrame` don't help me either.